enhancements/network/cloud-egress-ip.md

```diff
@@ -0,0 +1,468 @@
+---
+title: Cloud egress IP component
+authors:
+  - "@alexanderConstantinescu"
+reviewers:
+  - "@danwinship"
+  - "@abhat"
+  - "@squeed"
+  - "@tssurya"
+  - "@trozet"
+approvers:
+  - "@danwinship"
+  - "@abhat"
+  - "@squeed"
+creation-date: 2020-12-16
+last-updated: 2021-01-19
+status: implementable
+---
+
+# Cloud egress IP component
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+OVN-Kubernetes and openshift-sdn today support the feature egress IP on
```

Comment:
> but in the end are we going to implement the integration for sdn as well as in sdn is also part of the scope ?

Yes, this is what was decided by product management. 

>  I am mildly surprised since I thought sdn was .... dying.

Not for egress IP, it's one of the most sought after features in networking for most companies using OpenShift - hence why the exception.